src/test/java/io/debezium/connector/jdbc/JdbcSinkConnectorToMySQLIT.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.jdbc;
+
+import static io.debezium.connector.jdbc.SourcePipelineInvocationContextProvider.DATABASE_CONTAINER_NAME;
+import static io.debezium.connector.jdbc.SourcePipelineInvocationContextProvider.KAFKA_CONNECT_CONTAINER_NAME;
+import static io.debezium.connector.jdbc.SourcePipelineInvocationContextProvider.KAFKA_CONTAINER_NAME;
+import static io.debezium.connector.jdbc.SourcePipelineInvocationContextProvider.getNetwork;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.regex.Pattern;
+
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.junit.Assert;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestTemplate;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.KafkaContainer;
+import org.testcontainers.containers.MySQLContainer;
+import org.testcontainers.utility.DockerImageName;
+
+import io.debezium.testing.testcontainers.Connector;
+import io.debezium.testing.testcontainers.ConnectorConfiguration;
+import io.debezium.testing.testcontainers.DebeziumContainer;
+
+public class JdbcSinkConnectorToMySQLIT extends AbstractPipelineIT {
+    private static DockerImageName MYSQL_IMAGE = DockerImageName.parse("mysql:8.0");
+
+    private static MySQLContainer<?> sinkDatabaseContainer = new MySQLContainer<>(MYSQL_IMAGE).withNetwork(getNetwork())
+            .withDatabaseName("test");
+
+    @BeforeEach
+    void before() {
+        sinkDatabaseContainer.start();
+    }
+
+    @AfterEach
+    void after() {
+        sinkDatabaseContainer.stop();
+    }
+
+    @TestTemplate
+    void createSimplePipelineFromSourceToSink(Map<String, GenericContainer<?>> containerMap) throws InterruptedException {
```

Comment:
I don't quite get the test structure. I'd have expected something like:

* Write a DBZ change event to a topic
* Assert corresponding records in the sink database

Step 1 could be done in two variations: manually writing change events using a Kafka producer in the test, or via a Debezium source connector instance, triggered by writes to a source DB.

Reply:
Actually it quiet complicated than that, it has a testTemplate that tries to create a source connector for each Debezium source connector(right now only for MySql and Postgres) and then It tries to run SInkConnector and check it